src/client/dfuse/dfuse.h

```diff
@@ -541,6 +539,12 @@ struct entry_req {
 
 /* dfuse_inode.c */
 
+int
+dfuse_lookup_inode(struct dfuse_projection_info *fs_handle,
+		   struct dfuse_dfs *dfs,
+		   daos_obj_id_t *oid,
+		   ino_t *_ino);
```

Comment:
Why is it named _ino?

Reply:
This is a syntax I copied from @mchaarawi in dfs, where the _ prefix denotes out parameters that are set via pointers.